webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -214,43 +233,48 @@ describe('ContributionForm.vue', () => {
           postTitleInput = wrapper.find('.ds-input')
           postTitleInput.setValue(postTitle)
           await wrapper.vm.updateEditorContent(postContent)
-          categoryIds = ['cat12']
-          wrapper.find(CategoriesSelect).vm.$emit('updateCategories', categoryIds)
+          wrapper.find(CategoriesSelect).setData({ categories })
+          wrapper
+            .findAll('li')
+            .at(1)
+            .trigger('click') // language
+          await wrapper
+            .find(CategoriesSelect)
+            .findAll('button')
+            .at(1)
+            .trigger('click')
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-12T20:33:02Z" title="Tuesday, November 12th 2019, 9:33:02 pm +01:00">Nov 12, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -214,43 +233,48 @@ describe('ContributionForm.vue', () => {
           postTitleInput = wrapper.find('.ds-input')
           postTitleInput.setValue(postTitle)
           await wrapper.vm.updateEditorContent(postContent)
-          categoryIds = ['cat12']
-          wrapper.find(CategoriesSelect).vm.$emit('updateCategories', categoryIds)
+          wrapper.find(CategoriesSelect).setData({ categories })
+          wrapper
+            .findAll('li')
+            .at(1)
+            .trigger('click') // language
+          await wrapper
+            .find(CategoriesSelect)
+            .findAll('button')
+            .at(1)
+            .trigger('click')
```


I think we can improve this! what if someone adds more buttons? it would be nice to do similar to what @Mogge did here https://github.com/Human-Connection/Human-Connection/blob/master/webapp/components/FilterPosts/FilterPosts.spec.js#L114 ,don't you think?